src/tools/doall/src/DOALL_chunking.cpp

```diff
@@ -118,6 +118,107 @@ void DOALL::rewireLoopToIterateChunks(LoopDependenceInfo *LDI) {
     this->IVValueJustBeforeEnteringBody[ivPHI] = chunkedIVValues;
   }
 
+  auto sccManager = LDI->getSCCManager();
+  auto sccdag = sccManager->getSCCDAG();
+
+  /*
+   * Calculate chunking for periodic variable SCCs.
```

Comment:
Can you rewrite this comment to make it more precise?
For example, I don't think this code actually computes the chunking. I think what it does is generate the code to update the periodic variables to match the chunking strategy chosen by DOALL.

Reply:
changed in [c1a125b](https://github.com/arcana-lab/noelle/pull/74/commits/c1a125b1fc1a14142b08dffc099f767c2eb9845b)